index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Asha</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css">
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+</head>
+<body>
+    <div class="toolbar">
+        <h3 class="logo">My logo</h3>
+        <i class="fas fa-bars"></i>
+    </div>
+        <div class="Ellipse11"><img src="\Assets\Ellipse 11.png"></div>
+        <div class="Ellipse3"><img src="\Assets\Ellipse 3.png"></div>
+        <div class="Rectangle4"><img src="\Assets\Rectangle 4.png"></div>
+        <div class="Rectangle7"><img src="\Assets\Rectangle 7.png"></div>
+        <div class="Ellipse5"><img src="\Assets\Ellipse 5.png"></div>
+        <div class="Ellipse8"><img src="\Assets\Ellipse 8.png"></div>
+        <section>
+            <h2 class="Right Intro">Namaste!<br> I'm Asha</h2>
+            <p class="description">I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        </section>
+        <div class="Right contact">
+            <p class="media-text">LET’S CONNECT</p>
+        </div>
+        <div class="Right icons">            
+            <a href="https://twitter.com/asha_cep" target="_blank" class="media-link" rel="noopener"><i class='fab fa-twitter'></i></a> 
+            <a href="https://www.linkedin.com/in/asha-vijayan-67179a18" target="_blank" class="media-link" rel="noopener"><i class='fab fa-linkedin'></i></a>
+            <a href="https://medium.com/" target="_blank" class="media-link" rel="noopener"><i class='fab fa-medium-m'></i></a>
+            <a href="https://github.com/asharanjith" target="_blank" class="media-link" rel="noopener"><i class='fab fa-github'></i></a>
+            <a href="https://angel.co/asha-sethu-vijayan" target="_blank" class="media-link" rel="noopener"><i class='fab fa-angellist'></i></a>
+        </div>
+        <div class="Ellipse12"><img src="\Assets\Ellipse 12.png"></div>
+        <div class="Ellipse13"><img src="\Assets\Ellipse 13.png"></div>
+        <div class="Ellipse4"><img src="\Assets\Ellipse 4.png"></div>
```

Comment:
Kindly remove these elements, decorations are always background images, not img tags. Please use CSS to add this as background. :recycle: